README.md

```diff
@@ -0,0 +1,56 @@
+# Bitmark SDK for Java
```

Comment:
I think we should put README for each module in project. This file is just for Java SDK, how about others?